src/main/java/uk/gov/pay/ledger/transaction/search/common/TransactionSearchParams.java

```diff
@@ -169,6 +168,59 @@ public void setDisplaySize(Long displaySize) {
         if (isNotBlank(cardHolderName)) {
             filters.add(" lower(t.cardholder_name) LIKE lower(:" + CARDHOLDER_NAME_FIELD + ")");
         }
+        if (cardBrands != null && !cardBrands.isEmpty()) {
+            filters.add(" t.card_brand IN (<" + CARD_BRAND_FIELD + ">)");
+        }
+        if (isNotBlank(firstDigitsCardNumber)) {
+            filters.add(" t.first_digits_card_number = :" + FIRST_DIGITS_CARD_NUMBER_FIELD);
+        }
```

Comment:
This is already in `addCommonFilterTemplates`.
What about `lastDigitsCardNumber` though?

Reply:
Supposed to be `lastDigitsCardNumber`. Updated